components/Footer/Footer.js

```diff
@@ -0,0 +1,78 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function Footer() {
+  return (
+    <>
```

Comment:
You don't need a fragment `<>`element at this point. The `<StyledFooterList>`already nest the other tags.

Reply:
Thanks! I made the adjustment.